rdbms/models/addresses.py

```diff
@@ -0,0 +1,48 @@
+from sqlalchemy import Column
+from sqlalchemy import Integer
+from sqlalchemy import String
+from sqlalchemy import ForeignKey
+from sqlalchemy import DateTime
+from mimesis import Address as MimAddress
+from mimesis import Datetime
+
+from rdbms.models.database import Session
+
+from .database import Base
+
+
+class Address(Base):
+    __tablename__ = 'addresses'
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer, ForeignKey('users.id'))
+    street_address = Column(String)
+    street_address_two = Column(String)
+    city = Column(String)
+    state = Column(String)
+    postal_code = Column(String)
+    country = Column(String)
+    start_date = Column(DateTime)
```

Comment:
Where time of day is irrelevant, it's better to use Date, but DateTime works, too. By chosing it as the data type of the field you're forcing users to specify a time they started or accept a default value.